Sources/SwiftKuery/AuxiliaryTable.swift

```diff
@@ -16,25 +16,90 @@
 
 // MARK: AuxiliaryTable
 
-/// Subclasses of the `AuxiliaryTable` class are describing a table that is used in WITH clauses.
+/**
+ Subclasses of the `AuxiliaryTable` class are describing a `Table` that is used in WITH clauses.
+ ### Usage Example: ###
+ In this example, an `AuxiliaryTable` class is defined containing two columns.
+ A `ToDoTable` (as defined in `Table`), a `PersonTable` (as defined in `Column`) and a connection instances are initialized.
+ An instance of the `AuxiliaryTable` is then initialised from `Column` instances in "todotable".
+ This `AuxiliaryTable` is then used to create an SQL WITH query.
+ A description of the created query is then printed.
+ ```swift
+ class AuxTable: AuxiliaryTable {
+    let tableName = "aux_table"
+    let name = Column("name")
+    let finished = Column("finished")
+ }
+ 
+ let todotable = ToDoTable()          // ToDoTable() is a previously defined `Table` class
+ let persontable = PersonTable()      // PersonTable() is a previously defined `Table` class
+ let connection = createConnection()
+ 
+ let withTable = AuxTable(as: Select(todotable.toDo_completed.as("finished"), todotable.toDo_title.as("name"), from: todotable))
+ let withQuery = with(withTable, Select(withTable.finished, persontable.monthlyPay, from: persontable).join(withTable).on(persontable.name == withTable.name))
+ let stringQuery = connection.descriptionOf(query: withQuery)
+ print(stringQuery)
+ // Prints WITH aux_table AS (SELECT todotable.toDo_completed AS finished, todotable.toDo_title AS name FROM todotable) SELECT aux_table.name, persontable.monthlyPay FROM persontable JOIN aux_table ON persontable.name = aux_table.name"
```

Comment:
There are double quotes on the end of this Prints line which don't seem to match any other double quotes?

Reply:
Please can you check the whole string output as I got the following - I know we are likely to streamline `firstName` and `name` but I have an `aux_table.finished` where you have `aux_table.name` which doesn't look right:

Optional("WITH aux_table AS (SELECT toDoTable.toDo_completed AS finished, toDoTable.toDo_title AS name FROM toDoTable) SELECT aux_table.finished, personTable.monthlyPay FROM personTable JOIN aux_table ON personTable.firstName = aux_table.name")